CONTRIBUTING.md

```diff
@@ -1,4 +1,9 @@
-# Request for contributions
+# Contributing to OSEM
+Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) and send a PR!
```

Comment:
It is a little bit confusing as it seems that you need to create a issue to open a PR. I would also prefer not ot abbreviate PR. what about this:

Our open source project is open to contributions. Open your [issue](https://github.com/openSUSE/osem/issues/new) or work in one if the [open issues](https://github.com/openSUSE/osem/issues). Then open a Pull Request!